archiver/archiver_test.go

```diff
@@ -120,10 +114,7 @@ func TestJobsArchival(t *testing.T) {
 				},
 			},
 			Preferences: backendconfig.StoragePreferences{
-				GatewayDumps:     true,
-				BatchRouterDumps: true,
-				RouterDumps:      true,
-				ProcErrorDumps:   true,
+				GatewayDumps: true,
```

Comment:
I'm not very familiar with this bit. How come you're removing for example `RouterDumps` but not `GatewayDumps`? 🤔